app/templates/gentelella/users/events/index.html

```diff
@@ -90,5 +90,44 @@
     <script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/bootstrap-progressbar/bootstrap-progressbar.min.js') }}"></script>
+    <script type="text/javascript">
```

Comment:
@Himanshi-Khandelwal that was part of a previous PR .. In this PR, the only change done is supposed to be adjusting the logo width and the text underneath it